test/node-unit/mocha.spec.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var Mocha = require('../../lib/mocha');
+
+describe('.unloadFile()', function() {
+  it('should load and unload file to/from cache', function() {
+    var mocha = new Mocha({});
+    var filePath = __filename;
+    mocha.addFile(filePath);
+    mocha.loadFiles();
+
+    expect(
```

Comment:
I think you can write something like this, which IMO is more appropriate:

```js
expect(require.cache, 'to have property', require.resolve(filePath));
```

then later

```js
expect(require.cache, 'not to have property', require.resolve(filePath));
```